kalite/updates/api_views.py

```diff
@@ -129,10 +130,12 @@ def start_video_download(request):
     lang = json.loads(request.body or "{}").get("lang", "en")
 
     youtube_ids = get_download_youtube_ids(paths, language=lang)
-
+    ids_to_queue = {id_: title for (id_, title) in youtube_ids.iteritems()
```

Comment:
You are right, this is not the right fix. See the function above that retrieves the youtube_ids in the first place. Let's modify that instead. It is only used to retrieve ids for downloads, so you can add the filter there.

I would also suggest filtering on 'files_complete === 1', so that we fetch additional videos.

As noted by @radinamatic in a recent issue, we annotate videos as available if we have the English video + relevant language subtitle, so being 'available' is not a guarantee that the video has been downloaded.